__tests__/store/modules/inbox.spec.js

```diff
@@ -0,0 +1,134 @@
+import inbox from "../../../src/store/modules/inbox";
+import { createStore, useStore } from "vuex";
+import chatMessages from "../../../src/store/modules/chatMessages";
+jest.mock("../../../src/store/modules/chatMessages");
+import emails from "../../../src/store/modules/emails";
+jest.mock("../../../src/store/modules/emails");
+
+describe("inbox getters", () => {
+  it("getInboxItems", () => {
+    const store = createStore({
+      modules: {
+        inbox,
+        emails,
+        chatMessages,
+      },
+    });
```

Comment:
Seems like you could put this into a `beforeEach()` as you did for the second set of tests, since 4/5 tests in this section need this exact bit of code. The `isMobileDrawerOpen` test could then be done as a separate test.

Reply:
I was on the fence as you get more flexibility if you recreate it and some cases we could take advantage of it. However, since there is a preference I changed it. I made it a beforeAll so it is only created once. isMobileDrawerOpen doesn't use the store so it should be fine. The reason I had to use the store is because there are dependencies on other modules that break without it.